recipes/mender-artifact/rpm/mender-artifact.spec

```diff
@@ -0,0 +1,62 @@
+%define mender_artifact_version 4.1.0
+%define tag build1
+
+# We don't want debuginfo packages
+%global debug_package %{nil}
+
+Name: mender-artifact
+Summary: A CLI tool for working with Mender artifacts
+Version: %{mender_artifact_version}
+Release: 1%{?dist}
+License: Apache-2.0
+URL: https://github.com/mendersoftware/mender-artifact/
+
+# Once we have releases at GitHub not just tags, we should use a URL like the
+# one below, instead:
+# Source0: https://github.com/mendersoftware/mender-artifact/releases/download/...
+# The variant below, with the tarball name, results in a warning from rpmlint
+# that it's not a valid URL. But the URL to get it from looks like this:
+#   https://github.com/mendersoftware/mender-artifact/archive/refs/tags/4.1.0.tar.gz
+# which has no package name in tarball file name. So we use a custom file name
+# until we can use a release.
+Source0: %{name}-%{version}-%{tag}.tar.gz
```

Comment:
How will be build nightlies for the experimental repo? This is one of the point of high complexity in the debian recipes, where we have some crazy logic in the build script to generate valid versions out of Git info + CI pipeline info. See [here](https://github.com/mendersoftware/mender-dist-packages/blob/aa6c317b865dc019516df3f0187d80b26b2c43ec/mender-deb-package#L97).

Reply:
> How will be build nightlies for the experimental repo?

That's out of scope of this PR, really. First, I just want to have a spec file that can be used for building RPMs. Even if it was done manually on someone's laptop. Then we can have a repo (or repos). And only then we might consider if we need some repo(s) for nightlies as well.